src/main/java/kr/allcll/backend/admin/period/AdminPeriodApi.java

```diff
@@ -0,0 +1,31 @@
+package kr.allcll.backend.admin.period;
+
+import jakarta.servlet.http.HttpServletRequest;
+import kr.allcll.backend.admin.AdminRequestValidator;
+import kr.allcll.backend.admin.period.dto.PeriodRequest;
+import kr.allcll.backend.support.semester.Semester;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class AdminPeriodApi {
+
+    private final AdminPeriodService periodService;
+    private final AdminRequestValidator validator;
+
+    @PostMapping("/api/admin/service-period")
+    public ResponseEntity<Void> createPeriod(HttpServletRequest request,
+        @RequestParam Semester semesterCode,
+        @RequestBody PeriodRequest periodRequest) {
```

Comment:
저 미국가있을동안 추가된 코드를 훑어봤는데, 요 부분 컨벤션이 약간 흐트러진 것 같아요.
```suggestion
    public ResponseEntity<Void> createPeriod(
        HttpServletRequest request,
        @RequestParam Semester semesterCode,
        @RequestBody PeriodRequest periodRequest) {
```
저희 파라미터가 3개 이상이거나 너무 길어지게되면 전부 개행하기로 했던 것 같은데, 어떻게 생각하시나요?
기존 해윤님 금서님이 처음에 온보딩하시구 작성하셨던 `TimeTableApi`쪽을 봐도 3개 이상이면 개행을 전부 해주는 방식을 택했던 것 같아요!